closed/OpenJ9.gmk

```diff
@@ -185,6 +185,7 @@ $(foreach file, \
 		testlibA \
 		testlibB \
 		vmruntimestateagent29 \
+		clinkerffitests \
```

Comment:
Please move this up so the names remain sorted.

Reply:
Fixed.